2018q3/wiki.md

```diff
@@ -0,0 +1,38 @@
+## Various Documentation Bits ##
+
+Contact: 
+- Wiki:
+  - Kubilay Kocak, <koobs@FreeBSD.org>
```

Comment:
Replace individual wiki contacts with:

 * wiki-admin@f.o
 * #freebsd-wiki @ freenode IRC (bonus points if you link it with ircs:// url)